lib/domain/usecase/phonebook_contact_interactor.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:collection/collection.dart';
+import 'package:dartz/dartz.dart';
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/app_state/success.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/domain/app_state/contact/get_phonebook_contacts_state.dart';
+import 'package:fluffychat/domain/model/contact/contact_status.dart';
+import 'package:fluffychat/domain/model/contact/lookup_list_mxid_request.dart';
+import 'package:fluffychat/domain/repository/lookup_repository.dart';
+import 'package:fluffychat/domain/repository/phonebook_contact_repository.dart';
+import 'package:flutter/foundation.dart';
+import 'package:matrix/matrix.dart';
+
+class PhonebookContactInteractor {
+  static const int _progressStep = 10;
+  static const int _progressMax = 100;
```

Comment:
Can you explain more about this interactor and these constant

Reply:
![image](https://github.com/linagora/twake-on-matrix/assets/12546908/104abff6-d198-478a-8306-e44852c5cfba)
progress for that UI